doc/article/technical_article.tex

```diff
@@ -330,6 +330,19 @@ \subsection{User Interface}
 
 All time entries will be displayed on the time entry page.
 
+\section{Working with Scrum}
+Scrum describes a framework for agile software development. Agile in this
+context means, that requirements and results can evolve during the development.
+The development team uses Scrum for developing trckr and divides the tasks in
+sprints of two weeks. The team holds a short meeting every two days per week,
+where every team member presents completed and ongoing tasks. Every two weeks
+there is a sprint review, where the past and upcoming development phase gets
+reviewed and planned. Also, every team member is asked to talk about positive
+and negative experiences during the last sprint. This is called the sprint
+retrospective. Finally, the upcoming sprint has to be planned by the team. Every
+team member guesses an effort of time for the tasks the Scrum master (the
```

Comment:
estimate instead of guess
"Every team member estimates the effort for each task the Scrum master (the person that leads Scrum) chose for the upcoming sprint."